sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -231,6 +241,32 @@ private [sql] case class CreateTempTableUsing(
   }
 }
 
+private[sql] case class DropTable(
+    tableName: String,
+    isExists: Boolean,
+    temporary: Boolean) extends Command
```

Comment:
You don't need this class. All `RunnableCommand` are automatically wrapped with `ExecutedCommand` in `SparkStrategies.BasicOperators`.